sdk/core/az_core/inc/az_log.h

```diff
@@ -55,6 +55,9 @@ typedef enum
 
   AZ_LOG_IOT_SAS_TOKEN
   = _az_LOG_MAKE_CLASSIFICATION(_az_FACILITY_IOT, 2), ///< IoT Client generated new SAS token.
+
+  AZ_LOG_IOT_CONVENIENCE_LAYER
+  = _az_LOG_MAKE_CLASSIFICATION(_az_FACILITY_IOT, 3), ///< Azure IoT platform specific convenience layer.
```

Comment:
Can you please share a sample usage code/scenario that requires this log classification, to help me understand the motivation?

Reply:
This will be used in Azure RTOS convenience layer implementation https://github.com/azure-rtos/azure-iot-preview/blob/master/azure_iot/nx_azure_iot_hub_client.c